[Docker] Cannot find module 'dotenv'

- `git pull` latest commit on SSDNode cloud server
- `docker-compose up -d --force-recreate`
- `web-yjs`,`web-api`,`spawner-docker` complains about `Error: Cannot find module 'dotenv'`
- Tried `pnpm add dotenv`, then `docker-compose up -d --force-recreate`, the complain persists.

===========================
[INFO] 19:20:42 ts-node-dev ver. 2.0.0 (using ts-node ver. 10.9.1, typescript ver. 4.8.4)
Error: Cannot find module 'dotenv'
Require stack:
- /codepod/apps/web-yjs/src/run.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Function.Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/codepod/apps/web-yjs/src/run.ts:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._compile (/codepod/node_modules/.pnpm/source-map-support@0.5.21/node_modules/source-map-support/source-map-support.js:568:25)
    at Module.m._compile (/tmp/ts-node-dev-hook-45905237892175865.js:69:33)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at require.extensions..jsx.require.extensions..js (/tmp/ts-node-dev-hook-45905237892175865.js:114:20)
[ERROR] 19:21:13 Error: Cannot find module 'dotenv'
Require stack:
- /codepod/apps/web-yjs/src/run.ts

--- a/apps/desktop-api/package.json
+++ b/apps/desktop-api/package.json
@@ -18,7 +18,6 @@
     "apollo-server-express": "3.10.2",
     "bcryptjs": "^2.4.3",
     "dockerode": "^3.3.1",
-    "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "google-auth-library": "^8.7.0",
     "graphql": "16.6.0",

--- a/apps/spawner/src/run-docker.ts
+++ b/apps/spawner/src/run-docker.ts
@@ -2,8 +2,6 @@ import { startAPIServer } from "./server";
 
 import { killRuntime, spawnRuntime } from "./spawner-docker";
 
-require("dotenv").config();
-
 if (!process.env.JWT_SECRET) {
   throw new Error("JWT_SECRET env variable is not set.");
 }

--- a/apps/spawner/src/run-native.ts
+++ b/apps/spawner/src/run-native.ts
@@ -1,8 +1,6 @@
 import { startAPIServer } from "./server";
 import { killRuntime, spawnRuntime } from "./spawner-native";
 
-require("dotenv").config();
-
 if (!process.env.JWT_SECRET) {
   throw new Error("JWT_SECRET env variable is not set.");
 }

--- a/apps/web-api/package.json
+++ b/apps/web-api/package.json
@@ -18,7 +18,6 @@
     "apollo-server-express": "3.10.2",
     "bcryptjs": "^2.4.3",
     "dockerode": "^3.3.1",
-    "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "google-auth-library": "^8.7.0",
     "graphql": "16.6.0",

--- a/apps/web-api/src/server.ts
+++ b/apps/web-api/src/server.ts
@@ -15,9 +15,6 @@ import { typeDefs } from "./typedefs";
 import { createUserResolver } from "./resolver_user";
 import { RepoResolver } from "./resolver_repo";
 
-// dotenv
-require("dotenv").config();
-
 if (!process.env.JWT_SECRET) {
   throw new Error("JWT_SECRET env variable is not set.");
 }

--- a/apps/web-yjs/src/run.ts
+++ b/apps/web-yjs/src/run.ts
@@ -1,7 +1,5 @@
 import { startWsServer } from "./yjs-server";
 
-require("dotenv").config();
-
 if (!process.env.JWT_SECRET) {
   throw new Error("JWT_SECRET env variable is not set.");
 }

